cmd/krew/cmd/root.go

```diff
@@ -69,7 +79,9 @@ func init() {
 		paths.DownloadPath(),
 		paths.InstallPath(),
 		paths.BinPath(),
-		paths.InstallReceiptPath()); err != nil {
+	// TODO(corneliusweig): re-enable when migration code is removed:
+	//  paths.InstallReceiptPath(),
```

Comment:
I think you can still enable it.
We just need to find a different way to detect. An example could be: :the number of directories in `store/` are non-zero but `*.yaml`s in `receipts/` are zero" is cleaner and doesn't require you to do this.

Reply:
Yes, perfect. That'll do.